scripts/tests/core-integration.test.mjs

```diff
@@ -195,253 +190,81 @@ describe("CORE Integration Tests", () => {
-      await fs.writeFile(outputPath, content, "utf8");
-
-      const exists = await fs
-        .access(outputPath)
-        .then(() => true)
-        .catch(() => false);
-      expect(exists).toBe(true);
-
-      expect(path.basename(outputPath)).toBe("getting-started.md");
-    });
-
-    it("should create index.md for README files", async () => {
-      const content = "---\ntitle: Overview\n---\n\nOverview content...";
-      const outputPath = path.join(outputDir, "user-guide/index.md");
-
-      await fs.mkdir(path.dirname(outputPath), { recursive: true });
-      await fs.writeFile(outputPath, content, "utf8");
-
-      const exists = await fs
-        .access(outputPath)
-        .then(() => true)
-        .catch(() => false);
-      expect(exists).toBe(true);
-
-      expect(path.basename(outputPath)).toBe("index.md");
-    });
-
-    it("should create multiple output files for multiple source files", async () => {
-      const files = [
-        { source: "user-guide/getting-started.md", title: "Getting Started" },
-        { source: "user-guide/pepr-cli.md", title: "Pepr CLI" },
-        { source: "actions/mutate.md", title: "Mutate" },
-      ];
-
-      for (const file of files) {
-        const outputPath = path.join(outputDir, file.source);
-        const content = `---\ntitle: ${file.title}\n---\n\nContent...`;
-        await fs.mkdir(path.dirname(outputPath), { recursive: true });
-        await fs.writeFile(outputPath, content, "utf8");
-      }
-
-      const outputFiles = await glob(`${outputDir}/**/*.md`);
-      const relativePaths = outputFiles.map(f => path.relative(outputDir, f));
-
-      expect(relativePaths).toContain("user-guide/getting-started.md");
-      expect(relativePaths).toContain("user-guide/pepr-cli.md");
-      expect(relativePaths).toContain("actions/mutate.md");
-    });
-  });
-
-  describe("Complete Processing Simulation", () => {
-    it("should process file from docs/ through complete pipeline", async () => {
-      const sourceFile = path.join(docsDir, "010_user-guide/070_getting-started.md");
+  describe("Content Transformation Pipeline", () => {
+    it("should transform numbered paths and generate frontmatter in one pipeline", async () => {
+      const inputPath = "010_user-guide/070_getting-started.md";
       const sourceContent = "# Getting Started\n\nThis is a comprehensive guide...";
 
-      await fs.mkdir(path.dirname(sourceFile), { recursive: true });
-      await fs.writeFile(sourceFile, sourceContent, "utf8");
-
-      // Read source
-      const content = await fs.readFile(sourceFile, "utf8");
-
-      // Generate metadata
-      const relativePath = path.relative(docsDir, sourceFile);
+      const relativePath = path.relative(".", inputPath);
       const cleanPath = relativePath.replace(/\d+_/g, "");
 
-      // Generate frontmatter
-      const heading = content.match(/#[\s]+(.*)/);
+      const heading = sourceContent.match(/#[\s]+(.*)/);
```

Comment:
It looks like the value of this test has to do with ensuring the regex behaves as expected, right?